cli/sawtooth_cli/admin.py

```diff
@@ -22,15 +21,19 @@
 from sawtooth_cli.admin_command.utils import ensure_directory
 
 
-def do_admin(args):
+def do_admin(args, parser):
     data_dir = ensure_directory('data', '/var/lib/sawtooth')
 
+    if args.admin_cmd is None:
```

Comment:
I tested it. This condition does not get caught by argparse. At least that is what I determined empirically.